spec/requests/media_auth_request_spec.rb

```diff
@@ -2,17 +2,11 @@
 
 RSpec.describe "Authentication for Media requests", type: :request do
 
-  let(:allowed_loc) { 'ip.address1' }
   let(:user_no_loc_no_webauth) { User.new }
-  let(:user_no_loc_webauth) { User.new(webauth_user: true) }
```

Comment:
Thanks for that explanation.  I think I'm more comfortable now that I understand better what is tested where.  There is still a lot of magic in rails and can can that I don't always follow, and I think some of the original testing being removed here
- was about a then new action on a then new controller
- was about ensuring all the rails and cancan magic worked the way it's supposed to.

You are more conversant in both Rails and cancancancancancan so it probably made more sense to you from the get go.

I am now 👍 on this point being addressed.